recipes/kaleido-core/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "kaleido-core" %}
+{% set version = "0.1.0a3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/plotly/Kaleido/releases/download/v{{ version }}/kaleido_minimal_linux_x64.zip  # [linux]
+  sha256: 523fce13fca1fac2770273f2f19b3bac20d7a47e78ef9caa58590e98500535d8  # [linux]
+
+  url: https://github.com/plotly/Kaleido/releases/download/v{{ version }}/kaleido_mac.zip  # [osx]
+  sha256: 785b7a38ded6b5f525911f856eabf91cdc137550e224055b01707a73568aa508  # [osx]
+
+  url: https://github.com/plotly/Kaleido/releases/download/v{{ version }}/kaleido_win_x64.zip  # [win64]
+  sha256: 2bf0cea17642322aa8681c41f465550f2338afc68a03c4b241b88d823fefecd8  # [win64]
+
+build:
+  number: 0
+  ignore_run_exports:
+    # Chromium loads sqlite dynamically
+    - sqlite  # [linux]
+  missing_dso_whitelist:
+    - /usr/lib/*  # [osx]
+    - /System/Library/Frameworks/*  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    # Uncomment after feedstock creation to specify minimum GLIBC version and change
```

Comment:
Can you add a `skip: True # [linux]` to avoid uploading a package with wrong metadata?

Reply:
Good point. Done in ef7f6e7